src/hashes.js

```diff
@@ -7,8 +7,8 @@
 // V2 - { hash: hash_1, ...}
 //
 // This may change again with future contract versions
-function hash(project_id, token_id, token_hash) {
-  if (project_id < 3) {
+function hash(contract, token_id, token_hash) {
+  if (contract == "0x059edd72cd353df5106d2b9cc5ab83a52287ac3a") {
```

Comment:
Nit: it is probably worth adding a short documentation comment that specifies why the original AB contract is a special case here.

Reply:
Agreed, will make a note.